frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.tsx

```diff
@@ -81,6 +82,73 @@ const DataConsumptionCard: React.FC<DashboardItemProps> = ({
     if (suffixLabel) suffixLabel = `(in ${suffixLabel})`;
   }
 
+  if (isLoading) {
+    body = (
+      <>
+        <div className="skeleton-text nb-data-consumption-card__chart-skeleton" />
+        <GraphEmpty height={200} loading />
+        <div className="skeleton-text nb-data-consumption-card__chart-legend-skeleton" />
+      </>
+    );
+  } else if (!error && !chartData.some(_.isEmpty)) {
+    body = (
+      <>
+        <div className="nb-data-consumption-card__chart-label text-secondary">
+          {CHART_LABELS[sortByKpi]} {suffixLabel}
+        </div>
+        <Chart
+          containerComponent={
+            <ChartVoronoiContainer
+              labelComponent={<ChartTooltip style={{ fontSize: 8, paddingBottom: 0 }} />}
+              labels={({ datum }) => `${datum.y} ${maxUnit}`}
+              voronoiDimension="x"
+            />
+          }
+          minDomain={{ y: 0 }}
+          maxDomain={{ y: maxVal + 10 }}
```

Comment:
What do you suggest here ?
Showing plus 10 of max value seems good enough.

Reply:
The 3rd quartile.

Reply:
Why we should be using that ?

Reply:
how are we determining the `maxVal`? The third quartile is suitable for a tick point but not for the maxDomain. Basically you want the graph to be larger than the value by some factor. For large values `+10` would not make a difference. Maybe something like. `maxVal + maxVal*.25`

Reply:
1. `maxVal` if you check is the highest of all datapoints and its a humanized value.
2. Exactly this is not for tickpoints, but to have some extra space on axis for enhancing the look of graph.Typically above max value.
3. I like the suggestion here wrt `maxVal + maxVal*.25` just  one minor thing is that it will provide some fraction values and that can mess up in certain situations. (reason: d3 scale uses max domain to plot ticks and counts of ticks , and a fractional max domain does not comes up well, what I have tested)
I think `Math.round(maxVal + maxVal*.25)` would improve and save.

Reply:
3 sounds like the way to go. Thanks for the info. Dint know about the issue.